generated/generated.style

```diff
@@ -179,9 +179,15 @@ tags: inspire production Name: "PS.ProtectedSite"
 Title: en "Protected Sites"
 , de "Schutzgebiete"
 SpatialObjectTypes: http://inspire.ec.europa.eu/schemas/ps/4.0:ProtectedSite
+Styles: PS_ProtectedSite } Layer { id: "PS_ProtectedSite"
+registry-id: "http://inspire.ec.europa.eu/layer/PS.ProtectedSite"
+tags: inspire production Name: "PS.ProtectedSite"
+Title: en "Protected Sites"
+, de "Schutzgebiete"
+SpatialObjectTypes: http://inspire.ec.europa.eu/schemas/ps/5.0:ProtectedSite
 Styles: PS_ProtectedSite } Layer { id: "TN_AirTran"
 registry-id: "http://inspire.ec.europa.eu/layer/TN.AirTransportNetwork.AerodromeArea"
-tags: inspire production Name: "TN.AirTransportNetwork.AerodromeArea"
+tags: inspire Name: "TN.AirTransportNetwork.AerodromeArea"
```

Comment:
The production tag has been removed from the TN.AirTransportNetwork layer. Is that on purpose?